target/linux/mpc85xx/image/Makefile

```diff
@@ -67,7 +68,9 @@ endef
 TARGET_DEVICES += tl-wdr4900-v1
 
 define Device/red-15w-rev1
-  DEVICE_TITLE := Sophos RED 15w Rev.1
+  DEVICE_VENDOR := Sophos
+  DEVICE_MODEL := RED 15w
+  DEVICE_VARIANT := Rev.1
```

Comment:
v1? I do not find information of other revisions..

Reply:
There's only one (to my knowledge). However, the Rev. 1 is clearly labled on the device, so i would keep it.

Reply:
One could think about adding a space between Rev. and 1, but this is really a nitpick.

Reply:
It's indeed how SOPHOS has written it on the device.

Reply:
@blocktrron "Rev.1" or "Rev. 1"?

Reply:
"Rev.1"